### PR TITLE
feat(console): user icon

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

* add user icons
* refactor `ItemPreview` to support customize the icon style

## Linear Issue Reference

* LOG-2477

## Testing

image
image

### CODE REVIEW

**Comment by demonzoo** [comment]
Hey, shouldn't we hide the real names in order to protect our privacy?

File: packages/console/src/consts/avatars.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  gaodi,
+  hezhiyuan,
+  liming,
+  lisimeng,
+  niefenglong,
+  sungao,
+  wangsijie,
+  xiaoyijun,
+  yemingqing,
+  zhaochong,
```
[the remark this answers is not in the document]
Comment (demonzoo) [reply packages/console/src/consts/avatars.ts:22]: 🙅🏻‍♂️ pls dont

**Review by wangsijie — APPROVED** [review]
LGTM, please double check the size of images.

File: packages/console/src/consts/avatars.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/console/src/consts/avatars.ts]: per [discussion](https://silverhand-io.slack.com/archives/C0243P3E4KS/p1652842951524679), use number instead.

**Pull request merged**